src/components/DropzoneAreaBase.js

```diff
@@ -103,22 +111,52 @@ const defaultGetPreviewIcon = (fileObject, classes, isImage, titleBarTop) => {
 /**
  * This components creates a Material-UI Dropzone, with previews and snackbar notifications.
  */
-class DropzoneAreaBase extends React.PureComponent {
-    state = {
-        openSnackBar: false,
-        snackbarMessage: '',
-        snackbarVariant: 'success',
-    };
-
-    handleDropAccepted = async(acceptedFiles, evt) => {
-        const {fileObjects, filesLimit, getFileAddedMessage, getFileLimitExceedMessage, onAdd, onDrop} = this.props;
-
+const DropzoneAreaBase = ({
+    fileObjects,
+    filesLimit,
+    getFileAddedMessage,
+    getFileLimitExceedMessage,
+    getFileRemovedMessage,
+    getDropRejectMessage,
+    onAdd,
+    onAlert,
+    onDrop,
+    onDropRejected,
+    onDelete,
+    acceptedFiles,
+    alertSnackbarProps,
+    disableRejectionFeedback,
+    dropzoneClass,
+    dropzoneParagraphClass,
+    dropzoneProps,
+    dropzoneText,
+    getCols,
+    getPreviewIcon,
+    inputProps,
+    maxFileSize,
+    previewChipProps,
+    previewGridClasses,
+    previewGridProps,
+    previewText,
+    previewType,
+    showAlerts,
+    showFileNames,
+    useChipsForPreview,
+}) => {
+    const classes = useStyles();
+
+    const {
+        handleCloseSnackbar,
+        sendMessage,
+        snackbarMessage,
+        snackbarOpen,
+        snackbarVariant,
+    } = useSnackbar(onAlert);
```

Comment:
this is the way in which onAlert is now being used, by passing it in here we can call it with the snackbar